docs/notes/2.23.x.md

```diff
@@ -123,7 +125,9 @@ Support for including additional binaries in the pants sandbox through the `--go
 
 #### JVM
 
-When [the `jvm.reproducible_jars` flag](https://www.pantsbuild.org/2.21/reference/subsystems/jvm#reproducible_jars) is set resources jars are now also made reproducible, previously it was assumed resources jars are reproducible without additional action.
+When [the `jvm.reproducible_jars` flag](https://www.pantsbuild.org/2.23/reference/subsystems/jvm#reproducible_jars) is set resources jars are now also made reproducible, previously it was assumed resources jars are reproducible without additional action.
```

Comment:
Also took the opportunity to fix some links that were pointing to 2.21 references to point them to 2.23